package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Botpress, Inc.",
   "license": "AGPL-3.0",
   "scripts": {
-    "build": "yarn workspaces foreach --exclude @botpress/superinput -ptiv run build",
+    "build": "yarn workspaces foreach -ptiv run build",
     "watch": "yarn workspaces foreach --exclude @botpress/superinput -piv run watch",
     "clean": "yarn workspaces foreach --exclude @botpress/superinput -pi run clean",
```

Comment:
Can you also remove the `--exclude @botpress/superinput` for those two scripts too